spec/services/risk_assessment/high_risk_assessment_spec.rb

```diff
@@ -41,6 +41,15 @@
         end
       end
 
+      context 'when there is workitem without an uplift' do
```

Comment:
This spec is probably in the wrong place. This file is laid out rather unusually, but at the moment you'll see that you're inside the `context '#assess'`/`context 'returns true'` block, which shows a bunch of examples where the high risk assessment returns true. But this is an example where it returns false. So we probably want this inside the block that starts on line 72.

Reply:
Placed the spec in the `context '#assess'/ context 'returns false'` block. Updated the context claim with work item and existing spec for checking no uplift.